### PR TITLE
Add molecule tests for repositories_server and ssh_slave roles

### PR DESCRIPTION
**Pull request by osmocl**
Adding molecule test for remaining core roles:

- repositories_server 
- ssh_slave roles

The following roles cannot be tested in containers, due to limits of docker (`error: Device or resource busy`)
These roles will be tested in a full virtualized environment with kitchen.

- hosts_file
- set_hostname

### CODE REVIEW

File: roles/core/repositories_server/molecule/default/verify.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      command: firewall-cmd --zone=public --list-all
+      register: firewall_cmd_result
+      changed_when: False
+
+    - name: Firewall zone public check presence of service http
+      assert:
+        that: firewall_cmd_result.stdout |
+                    regex_findall(('http'), multiline=False) | length == 1
+
+    - name: Assert pxe needed packages are installed
```
Comment (btravouillon) [inline roles/core/repositories_server/molecule/default/verify.yml:23]: why 'pxe needed packages'?

File: roles/core/repositories_server/molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      assert:
+        that: "'httpd' in ansible_facts.packages"
+
+    - name: Retrieve file /etc/httpd/conf/httpd.conf system status
```
Comment (btravouillon) [inline roles/core/repositories_server/molecule/default/verify.yml:27]: The role does not manage this file, there is no need to test it.

File: .github/workflows/molecule-actions-core-repositories_server.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          - centos:7
+          - centos:8
+
+    env:
+      MOLECULE_DISTRO: ${{ matrix.distro }}
+      PY_COLORS: 1
+      ANSIBLE_FORCE_COLOR: 1
+    steps:
+      - name: Set INSTANCE_DISTRO variable
+        run: echo "::set-env name=INSTANCE_DISTRO::${MOLECULE_DISTRO/:/}"
```
Comment (btravouillon) [inline .github/workflows/molecule-actions-core-repositories_server.yml:32]: I don't see any usage of INSTANCE_DISTRO in the test. Do you need this env variable?

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍
Thanks!

**Comment by oxedions** [comment]
Dear @osmocl 

Thanks a lot for this PR.

Merging.

**Pull request merged**